### ISSUE TITLE
Lock view to axis/pixels

### ISSUE THREAD

**Issue by clbarnes** [saalfeldlab/paintera#391]
Treating the data as it really is (a sampling of a real-world space) is useful for viewing and slicing, but for painting at the same resolution as the raw image, especially in anisotropic data, it can be helpful to keep the orthoviews locked to the axes of the data, and to ensure that the crosshair always snaps to a pixel. Scrolling through e.g. the sectioning axis of ssTEM data is hard work otherwise: skimming through quickly is possible with the fast scroll shortcut, but going through one section at a time for painting is difficult.

Would a "locked view" mode be possible to implement? Or are there other design concerns?

**Comment by hanslovsky** [comment]
You can toggle rotations in the preference pane (toggle with `P`) with the `Rotations` check box at `Settings>Navigation`. If you toggle rotations off, the viewer will be locked in the current location but you can still zoom and translate. Use `shift + Z` to align the viewer axes with the arbitrary "global" coordinate system. Side note: It is still possible to navigate to bookmarks (which remember the rotation) and to align viewer axes (`shift + Z`) even if you toggle rotations off.

Regarding the scroll speed: We could expose the scroll distance as a parameter for the user to specify in the settings.

**Comment by hanslovsky** [comment]
A possible setting would be "snap to voxel grid if aligned" that would snap the viewer plane to the nearest data plane when navigating and aligned with the data axes.

**Comment by clbarnes** [comment]
> You can toggle rotations in the preference pane

Perfect! Sorry I missed that.

> A possible setting would be "snap to voxel grid if aligned"

That would be really helpful! The alternative (which I did with bigCAT) was to try to figure out what zoom level makes one scroll approximate one z slice, which is a pain.

**Comment by constantinpape** [comment]
> > A possible setting would be "snap to voxel grid if aligned"
> 

I agree, this would be a really useful feature to make painting in consecutive slices much easier.

**Comment by hanslovsky** [comment]
cc @igorpisarev and @axtimwalde as I will likely not be able to address this in my remaining days at Janelia
